recipes/easydev/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set version = "0.9.31" %}
+{% set name = "easydev" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/e8/6a/edeff86ca5fe5250d96523d4bc2d886b2c37a3ef4341f22ab4e2b270d121/{{ name }}-{{ version }}.tar.gz
+  md5: 68353d68eb7249269c7f8b1033e56ac3
```

Comment:
Please update the section above this comment to match the [`example`]( https://github.com/conda-forge/staged-recipes/blob/e6bcccf6a72c3ca4411817f89e729c951a048462/recipes/example/meta.yaml#L1-L19 ).

Note: [pypi.org]( https://pypi.org/ ) provides `sha256`.